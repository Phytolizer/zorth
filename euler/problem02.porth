include "std.porth"

macro acc mem end

1 2 while over 4000000 < do
  over 2 mod 0 = if
    over acc load64 + acc swap store64
  end

  swap over +
end
acc load64 print
