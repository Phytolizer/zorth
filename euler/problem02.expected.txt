: returncode 0
: stdout 8
4613732

: stderr 0

